native_client/ctcdecode/ctc_beam_search_decoder.cpp

```diff
@@ -36,7 +36,7 @@ DecoderState::init(const Alphabet& alphabet,
   prefix_root_.reset(root);
   prefixes_.push_back(root);
 
-  if (ext_scorer != nullptr) {
+  if (ext_scorer != nullptr && (bool)ext_scorer_->dictionary) {
```

Comment:
int: I usually prefer `(bool)(ext_scorer_->dictionary)` for ease of reading

Reply:
Done.